config/default.yml

```diff
@@ -334,3 +334,8 @@ FactoryGirl/DynamicAttributeDefinedStatically:
   Description: Prefer declaring dynamic attribute values in a block.
   Enabled: true
   StyleGuide: http://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/FactoryGirl/DynamicAttributeDefinedStatically
+
+Capybara/CurrentPathExpectation:
```

Comment:
This is a semi-alphabetical list (`RSpec/` is before `Capybara/`) – could we move this entry to line 328?